src/main/scala/org/broadinstitute/dsde/workbench/leonardo/auth/WhitelistAuthProvider.scala

```diff
@@ -15,30 +16,48 @@ class WhitelistAuthProvider(config: Config, serviceAccountProvider: ServiceAccou
 
   val whitelist = config.as[Set[String]]("whitelist").map(_.toLowerCase)
```

Comment:
Can we delete `WhitelistAuthProvider` or are we keeping it because it might be useful for someone who wants to implement a whitelist for their own Leo instance in the future?

Reply:
I'd say the latter. It's a simple enough example to point people at and plausibly provides some value.

Reply:
I think the Verily-internal people are actually using it. I was even thinking we should add some unit tests for it (similar to SamAuthProviderSpec).

Reply:
I think the Verily-internal people are actually using it. I was even thinking we should add some unit tests for it (similar to SamAuthProviderSpec).

Reply:
Turns out `LeonardoServiceSpec` uses `WhitelistAuthProvider` so it's probably covered in tests.